.github/workflows/build-deb.yaml

```diff
@@ -0,0 +1,85 @@
+name: Build debian packages
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+
+env:
+  UBUNTU_VERSIONS: '["noble", "devel"]'
+
+jobs:
+  define-versions:
+    name: Define build versions
+    runs-on: ubuntu-latest
+    outputs:
+      ubuntu-versions: ${{ env.UBUNTU_VERSIONS }}
+    steps:
+      - run: 'true'
+
+  build-deb-package:
+    name: Build ubuntu package
+    runs-on: ubuntu-latest
+    needs: define-versions
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu-version: ${{ fromJSON(needs.define-versions.outputs.ubuntu-versions) }}
+    outputs:
+      run-id: ${{ github.run_id }}
+      pkg-name: ${{ env.PKG_NAME }}
+      pkg-version: ${{ env.PKG_VERSION }}
+      # FIXME: use this when https://github.com/canonical/desktop-engineering/pull/58 is in
+      # pkg-src-changes: ${{ env.PKG_SOURCE_CHANGES }}
+
+    steps:
+      - name: Checkout authd code
+        uses: actions/checkout@v4
+
+      - name: Build debian packages and sources
+        uses: canonical/desktop-engineering/gh-actions/common/build-debian@main
+        with:
+          docker-image: ubuntu:${{ matrix.ubuntu-version }}
+          extra-source-build-deps: |
+            ca-certificates
+            git
+            libssl-dev
+          extra-source-build-script: |
+            cargo install --locked --root=/usr cargo-vendor-filterer
+            command -v cargo-vendor-filterer
+
+  run-autopkgtests:
+    name: Run autopkgtests
+    runs-on: ubuntu-latest
+    needs:
+      - define-versions
+      - build-deb-package
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu-version: ${{ fromJSON(needs.define-versions.outputs.ubuntu-versions) }}
+
+    if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') ||
+       contains(join(github.event.commits.*.modified), 'debian/') ||
+       contains(join(github.event.commits.*.modified), '.github/workflows/build-deb.yaml') ||
+       startsWith(github.ref, 'refs/tags/') ||
+       github.event_name == 'release' }}
+
+    steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v4
+      with:
+        run-id: ${{ needs.build-deb-package.outputs.run-id }}
+        merge-multiple: true
```

Comment:
See my comment on the merge multiple than on the other PR.

Reply:
This is solved now I guess (as it was in the other PR)